boards/matek/h743/init/rc.board_sensors

```diff
@@ -20,3 +20,12 @@ dps310 -I start -a 118
 
 # External mag
 qmc5883l -X start -a 13
+
+# External airspeed sensor/differential pressure
+if ! ms4525do -I start -a 40
```

Comment:
This is a separate external module you're plugging in?

Reply:
mRo I2C Airspeed Sensor JST-GH - MS4525DO
https://store.mrobotics.io/mRo-I2C-Airspeed-Sensor-JST-GH-p/m10030a.htm
![photo_2022-05-06_23-17-06](https://user-images.githubusercontent.com/64059887/167162358-377ce81f-d942-4385-b4cd-99922b7df30c.jpg)

Usually the I2C2 which is internal is used for airspeed sensor (according to Matek website Ardupilot and Inav recommandations)
![Screenshot 2022-05-06 23:19:09](https://user-images.githubusercontent.com/64059887/167162837-e05ec252-a27f-46de-afd8-d0f510726baa.png)

Reply:
That's fine, I just don't think it should be included like this if it's not physically part of the board.

Can you try starting it with the parameter? https://github.com/PX4/PX4-Autopilot/blob/90789be68f9d6c3c74597652b7a2681f4e3d2c28/ROMFS/px4fmu_common/init.d/rc.sensors#L145-L149

Reply:
fixed with [0c9f6ac](https://github.com/matthewoots/PX4-Autopilot/commit/0c9f6acc4828159cec01f01d4a0ebda59d29fae9)